### ISSUE TITLE
Wind speed animation

### ISSUE THREAD

**Issue by papjul** [breezy-weather/breezy-weather#958]
### Discussed in https://github.com/breezy-weather/breezy-weather/discussions/313

<div type='discussions-op-text'>

<sup>Originally posted by **tjsuominen** August  6, 2023</sup>
I am using Today Weather in parallel with Breezy Weather (Love your app, keep up the great work!).

One thing that I would love to get "adopted" somehow from TW is the animation effect of wind speed (both for the summary and as separate card)

Additional note: TW does not have a seasonal visualization in the wind speed animation.

Why? Because for most people showing the single "Wind x.x" speed does not tell much, but when it gets animated in a neat way, it tells a lot more.

I'm not in favor of "copying" features, but I hope that could be implemented in some neat way.

Any additional thoughts?
</div>

**Comment by papjul** [comment]
Open to contributions, but I'm not sure where we should put this animation, if you have any idea? A dedicated card would be too much IMO

**Comment by min7-i** [comment]
For inspiration, the [Yr app](https://play.google.com/store/apps/details?id=no.nrk.yr) has a wind speed animation integrated into their wind icon (see top right in the recording below).

https://github.com/breezy-weather/breezy-weather/assets/97251923/dcc62c59-4903-4708-a1ec-2233b5ebce7e

Instead of creating a "large" animation like Today Weather does, we could adapt Yr's approach for our wind icon in the header. Although I'm not sure if I like the idea of having a single animated icon while the rest of the header icons is static.
<img src="https://github.com/breezy-weather/breezy-weather/assets/97251923/d9a7b501-4803-4e9b-b987-6ac99f6d5812" width=35%>

**Comment by papjul** [comment]
I had something like that in mind, but I wasn't sure how it would be designed. Nice to see this example!

And it also fixes #738, but our icon is a bit small in comparison.

We could also add it to the hourly details dialog, and daily details activity.

> I'm not sure if I like the idea of having a single animated icon while the rest of the header icons is static

It could be animated only when you tap on it, like we do when you tap on the weather icon on hourly forecast.
